### ISSUE TITLE
Grunt 0.4 Release

### ISSUE THREAD

**Issue by tkellen** [phated/grunt-jade#16]
I'm posting this issue to let you know that we will be publishing Grunt 0.4 on Monday, February 18th.

If your plugin is not already Grunt 0.4 compatible, would you please consider updating it? For an overview of what's changed, please see our [migration guide](https://github.com/gruntjs/grunt/wiki/Upgrading-from-0.3-to-0.4).

If you'd like to develop against the final version of Grunt before Monday, please specify `"grunt": "0.4.0rc8"` as a devDependency in your project. After Monday's release, you'll be able to use `"grunt": "~0.4.0"` to actually publish your plugin. If you depend on any plugins from the grunt-contrib series, please see our [list of release candidates](https://github.com/gruntjs/grunt/wiki/Plugin-Release-Candidate-Versions) for compatible versions. All of these will be updated to final status when Grunt 0.4 is published.

Also, in an effort to reduce duplication of effort and fragmentation in the developer community, could you review the [grunt-contrib](http://github.com/gruntjs) series of plugins to see if any of your functionality overlaps significantly with them?  Grunt-contrib is community maintained with 40+ contributors&mdash;we'd love to discuss any additions you'd like to make.

Finally, we're working on a new task format that doesn't depend on Grunt: it's called [node-task](/tkellen/node-task).  Once this is complete, there will be one more conversion, and then we'll never ask you to upgrade your plugins to support our changes again.  Until that happens, thanks for bearing with us!

If you have any questions about how to proceed, please respond here, or join us in [#grunt on irc.freenode.net](irc://irc.freenode.net/#grunt).

Thanks, we really appreciate your work!

**Comment by JoeStanton** [comment]
:+1: Would like to see Grunt 0.4.x compatibility!

**Comment by tkellen** [comment]
Would **love** to see any features this provides that [grunt-contrib-jade](/gruntjs/grunt-contrib-jade) doesn't merged in!

**Comment by phated** [comment]
We just launched our game engine, so I should be able to progress on this sometime this week (hopefully tomorrow). Will look into the amount of work would go into adding the features to grunt-contrib-jade

**Comment by tkellen** [comment]
Awesome.  Thanks Blaine.

**Comment by phated** [comment]
Got this plugin working with Grunt `~0.4.0`, but will leave this open until I look into adding features to grunt-contrib-jade

**Comment by phated** [comment]
@tkellen started digging into grunt-contrib-jade code, but then had the brilliant idea to go look at the issues and PRs.  It looks like someone else is trying to implement these same features in some open PRs.  I don't feel like trying to re-implement the same features.  The grunt-contrib-jade code would need to be reworked quite a bit to make all of my features work and doesn't seem to get much love.  If you have a solution to these concerns, let me know and I will take a look into helping you with that plugin.
